### ISSUE TITLE
Check for `allow_module_level=True` in module-level `pytest.skip`

### ISSUE THREAD

**Issue by sobolevn** [m-burst/flake8-pytest-style#188]
# Rule request

## Description

Right now this is not detected as an error:

```python
import pytest

pytest.skip('Ignore this module')
```

## Rationale

This is also an error with `pytest`:

```
_____________ ERROR collecting tests/test_multiprocess_aioredis.py _____________
Using pytest.skip outside of a test will skip the entire module. If that's your intention, pass `allow_module_level=True`. If you want to skip a specific test or an entire class, use the @pytest.mark.skip or @pytest.mark.skipif decorators.
```

**Comment by m-burst** [comment]
Hi @sobolevn,
I'm not sure that duplicating `pytest`'s runtime checks on the linter level is the way to go.
Linters and tests are typically run in the same CI pipeline, so this duplication will only lead to extra work (both for me as the developer of the plugin and for the interpreter in runtime).
What do you think?
